docs/release-notes/canceled-experiment-trial-fix.txt

```diff
@@ -0,0 +1,6 @@
+:orphan:
+
+**Fixes**
+
+-  Experiments with a single trial are now considered canceled when their trial is
```

Comment:
will that also be now true for HP search experiments where a user kills all trials as they appear?

Reply:
yes.

Reply:
but they would need to kill them all